src/looprestoration.rs

```diff
@@ -3276,13 +3278,15 @@ mod neon {
     }
 }
 
+#[deny(unsafe_op_in_unsafe_fn)]
```

Comment:
This doesn't propagate from the module level? Thought it did.

Reply:
It does, but I've put them on all the `unsafe extern "C"` fallback `fn`s before making the entire `mod` safe, so I wanted to be consistent.